### DISCUSSION TITLE
Breadcrumb Hydration Error

### DISCUSSION THREAD

**Discussion by piyusharmap** [shadcn-ui/ui#4416] [Q&A]
As per the documentation, using Link inside BreadcrumbLink throws a hydration error. This happens because both BreadcrumbLink and Link are rendered as <a> tags, which results in nesting two <a> tags. To avoid this issue, you should omit the use of Link when using BreadcrumbLink.

![image](https://github.com/user-attachments/assets/7ed559f7-aaeb-45a5-8597-9285f4572cdb)

To solve this I simply used the href attribute inside the `BreadcrumbLink`.

![image](https://github.com/user-attachments/assets/81d8c01a-6c59-4b15-a93a-5b8afc5ae906)

**Comment by luan-coelho** [comment]
It is necessary to update the examples in the Breadcrumb documentation.

https://ui.shadcn.com/docs/components/breadcrumb#link-component

**Comment by luan-coelho** [comment]
Another way that also works is to omit the BreadcrumbLink component.

```jsx
<Breadcrumb>
      <BreadcrumbList>
        <BreadcrumbItem>
          <Link href="/">Home</Link>
        </BreadcrumbItem>
        <BreadcrumbSeparator />
        <BreadcrumbItem>
          <BreadcrumbPage>Breadcrumb</BreadcrumbPage>
        </BreadcrumbItem>
      </BreadcrumbList>
    </Breadcrumb>```
